CharacterRandomizer.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View } from 'react-native';
+import styled from 'styled-components/native';
+
+//Note! Some comments are in Swedish, beacuse this app is mainly focused on swedish rpg.
```

Comment:
Thank you for that comment, but it doesn't change anything - all comments and instructions should be written in English. We cannot predict who will use this code later on, it should be accessible to all developers, not only Swedish speakers